ext/native/thin3d/thin3d_vulkan.cpp

```diff
@@ -1467,6 +1469,8 @@ void CreateImage(VulkanContext *vulkan, VKImage &img, int width, int height, VkF
 	}
 	vkCreateImage(vulkan->GetDevice(), &ici, nullptr, &img.image);
 
+	// TODO: If available, use nVidia's VK_NV_dedicated_allocation for framebuffers
```

Comment:
I wonder if aliasing (not allowed by this extension) could be useful in some of the "depal as a different format" or "clear using 565" cases, though.  But yeah, probably better.

-[Unknown]